Examples/php/pragmas/runme.php

```diff
@@ -2,4 +2,5 @@
 
 require "example.php";
 
+echo "Version - " . ((new ReflectionExtension('example'))->getVersion());
 ?>
```

Comment:
Good, except this is an example, not part of the testsuite.  If someone in the future makes a change which breaks your new feature or it doesn't work correctly with a new version of PHP we won't get a test failure.  Running this example will print the wrong thing, but not flag an error.

We really want an automated test which will fail if the version reported via reflection doesn't match that set with the pragma - that should be done via the automated testsuite under `Examples/test-suite`.  There's support for language-specific tests there (and you'll need to get familiar with that testsuite for your gsoc project).

Having an automated feature test also gives a patch reviewer more confidence your implementation works as it should - CI runs the testsuite automatically on each PR for most languages (including PHP 5 and 7).